lib/vapor/watch.ex

```diff
@@ -10,20 +10,34 @@ defmodule Vapor.Watch do
   end
 
   def init(state) do
+    state = Map.merge(%{error_count: 0, current_alarm: false}, state)
     schedule_refresh(state.plan)
     {:ok, state}
   end
 
   def handle_info(:refresh, %{plan: plan, store: store, layer: layer}=state) do
     case Provider.load(plan.provider) do
       {:ok, new_config} ->
+        # We don't mind if we repeatedly spam the logs when there's a problem,
+        # but if everything is okay, we ought not attempt to clear an alarm
+        # that might not exist because the attempt with spam the logs.
```

Comment:
I'm not sure I understand the last line of this comment. I get the gist, but the last line is worded a little weird I think.

Reply:
Every time we clear the alarm, it gets logged (even if the alarm is not active). The comment was mostly to cover "why are we even keeping track of whether or not we have an alarm? Clearing an alarm is idempotent, let's just clear it..."